about.html

```diff
@@ -10,55 +10,91 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
+\
+
+<div class="container">   
+      <header class= "nav-bar">
+       <nav> 
+       <img class="logo" src="img/lambda-black.png" alt="Great Idea! Company logo.">
+           <a href="index.html">Services</a>
+           <a href="#">Product</a>
+           <a href="#">Vision</a>
+           <a href="#">Features</a>
+           <a href="about.html">About</a>
+           <a href="#">Contact</a>
+        </nav>
+          
+       </header>
+          
+     <body>
+      
+
+      <img class="logo" src="img/jumbo-about.png" alt="Great Idea! Company logo.">
+      
+     <div>
+      <h3>About<h3>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </div>
+    
+  
+  <section class= "topContent">
+
 
-<body>
-    <div class="container about-page">
+      <div class= "strategy info">
+        <img src="img/about-plan.png" alt="strategy">
+        <h3>Strategy</h3>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
```

Comment:
I like how you wrote these in their own self contained units with multiple classes for targeting in CSS